pkgs/development/tools/build-managers/scala-cli/default.nix

```
@@ -49,6 +49,15 @@ stdenv.mkDerivation {
     runHook postInstall
   '';
 
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+
+    $out/bin/scala-cli version --offline
+
+    runHook postInstallCheck
+  '';
```

I'd prefer something like the following, so that the version is actually tested:

```suggestion
  passthru.tests.version = testers.testVersion {
    package = scala-cli;
    command = "scala-cli version --offline";
  };
```

---

Thanks! That's pretty cool.